canonical-web-and-design/www.ubuntu.com
# Rebuild the openstack features page for ua4i

## Done
Rebuild the openstack features page for ua4i

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /openstack/features
- Check it matches [the copydoc](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit#)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5018
